### PR TITLE
feat(blocks): add more menu button on format bar

### PR DESCRIPTION
**Pull request by fundon**
Closes: [BS-814](https://linear.app/affine-design/issue/BS-814/在-format-bar-上-添加-more-menu-button)

Adds `more` menu button on the format-bar.

### Includes actions:

* copy
* duplicate
* delete

### Other:

* [x] tests

Screenshot 2024-07-09 at 16 36 34

### CODE REVIEW

File: tests/format-bar.spec.ts [hunk 1]
```diff
@@ -1650,3 +1631,65 @@ test('create linked doc from block selection with format bar', async ({
...
+
+    await focusRichText(page, 1);
+    await pasteByKeyboard(page);
+    await waitNextFrame(page);
+
+    await assertRichTexts(page, ['123', '456456', '789']);
+  });
+
+  // fixme: Copy and paste is broken(ctrl-c/ctrl-v).
+  test.skip('should be able to perform the duplicate action', async ({
```
Comment (fundon) [inline tests/format-bar.spec.ts:1658]: After https://github.com/toeverything/blocksuite/pull/7386, Copy and paste is broken. /cc @fourdim

Comment (fundon) [reply tests/format-bar.spec.ts:1658]: ### Before

https://github.com/toeverything/blocksuite/assets/27926/f48e9ae6-c414-4743-bd67-88a01d76d6c3


### After

https://github.com/toeverything/blocksuite/assets/27926/e00c8081-0094-4159-830b-a80290d39881

**Review by Saul-Mirone — COMMENTED** [review]
seems there're some conflicts

File: tests/format-bar.spec.ts (hunk 1 above)
Comment (Saul-Mirone) [reply tests/format-bar.spec.ts:1658]: @fourdim PTAL

**Pull request merged**